src/angularjs/src/styles/layout/_footer.scss

```diff
@@ -36,12 +36,13 @@ footer {
 }
 
 .stripe {
-  background-image: url(../assets/images/footer-stripes.png);
```

Comment:
I can't find any other references to `footer-stripes.png` remaining in the code. Shall we remove the `png` from assets as well?